web-animations-2/Overview.bs

```diff
@@ -339,6 +352,37 @@ descendant effects and custom effects such that the first condition is:
 >     effect</a> or executing any <a>custom effects</a> associated with an
 >     <a>animation effect</a>.
 
+<h4 id='the-current-phase-of-an-animation'>The current phase of an
+Animation</h4>
+
+In addition to their <a>current time</a>, <a>animations</a> provide their
+<a>associated effects</a> with a <dfn>current phase</dfn>.
+This is used in determining the animation effect's phase in certain boundary
+conditions.
+
+An <a>animation's</a> <a>current phase</a> has the same range of values as the
+[=timeline phase=].
+
+The <a>current phase</a> is calculated from the first matching condition from
+below:
+
+<div class="switch">
+
+:   If the animation's <a>hold phase</a> is set,
+::  The <a>current phase</a> is the animation's <a>hold phase</a>.
+
+:   If <em>either</em> of the following are true:
+
+    1.  the animation has no associated <a>timeline</a>, or
+    1.  the animation's [=start time=] is <a>unresolved</a>.
+
+::  The <a>current phase</a> is [=timeline inactive phase|inactive=].
+
+:   Otherwise,
+::  The <a>current phase</a> is the [=timeline phase=] of the associated
+    <a>timeline</a>.
```

Comment:
The animation effect handles that calculation, this is just the phase that gets passed to the animation effect to help it with its calculation. It is basically passing through the timeline phase, but adjusting for if the animation is paused.

Reply:
Consolidated by moving the logic to the effect phase calculation.  I think originally this was a bit confusing since when the timeline phase = active case, we still need to determine if the effect phase is before, active or after.  So timeline/animation phase is not the same as effect phase.  Having said that, I think the benefit of encapsulating hold phase is not worth the potential for confusing in introducing "animation phase".